src/DotVVM.Framework/Controls/Repeater.cs

```diff
@@ -97,7 +98,11 @@ public string WrapperTagName
         /// </summary>
         protected internal override void OnLoad(IDotvvmRequestContext context)
         {
-            SetChildren(context);
+            if (context.IsPostBack)
+            {
+                SetChildren(context, true);
```

Comment:
@exyi Do we need to populate `Children` in `OnLoad` when the request is not a postback?

Reply:
Yes we need, we need to build the tree in OnPreRender, it's required for `Postback.Update` to work and we need to render it in OnLoad, it's needed for invocation of a command in the children.

Reply:
Exactly. But it is enabled for postbacks. It can't break command invocation, can it?.

Reply:
@tomasherceg What do you think about this?